frontend/src/layout/Navbar.tsx

```diff
@@ -70,7 +70,7 @@ const Navbar = ({ currentLogin, logoutUser, loading }: Props) => {
         <Link to='/categories' className='btn btn-ghost btn-sm rounded-btn'>
           Categories
         </Link>
-        {currentLogin.user.is_admin && (
+        {currentLogin.user.is_admin === 1 && (
```

Comment:
if not admin what does it return?

Reply:
It leaves the '0' thing as shown earlier

Reply:
Try this if it works
```suggestion
        {!!currentLogin.user.is_admin && (
```